app/javascript/components/upload/CoordinateLabelStep.js

```diff
@@ -70,23 +69,26 @@ function CoordinateLabelForm({
             -10.68&#09;-52.64&#09;-57.34&#09;Region 2<br/>
             ...
           </pre>
-        <p>
-          <strong>These are not cluster files</strong> - they are annotations to overlay on top of a cluster.<br/>
-          The file must be a plain text (.txt) file with at least 3 columns and a header row containing the values <strong>X</strong>, <strong>Y</strong>, and <strong>LABELS</strong>.  The file may have an optional column of <strong>Z</strong> (for 3d clusters).
-          The last column must contain text labels to display at the specified coordinates.
-        </p>
-        <p><i className="fas fa-fw fa-exclamation-triangle text-warning"></i> The coordinates of the labels must fall inside the ranges of the cluster they are associated with for them to render.</p>
+          <p>
+            <strong>These are not cluster files</strong> - they are annotations to overlay on top of a cluster.<br/>
```

Comment:
Realizing this language is a bit ambiguous.
```suggestion
            <strong>These are not cluster files</strong> - they are annotations to overlay on top of a scatter plot.<br/>
```